From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: chiranz <chiranz08@users.noreply.github.com>
Date: Mon, 5 Oct 2026 16:14:54 +0000
Subject: [PATCH 13/13] Revert "ThemeOverlayController: black
 system_accent2_800 with the pure black theme (USB sheet, drawer with blur
 off)"

This reverts commit 4b0176f227ccf1c4d58db5004ec7b656e39678da.
---
 .../android/systemui/theme/ThemeOverlayController.java   | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/theme/ThemeOverlayController.java b/packages/SystemUI/src/com/android/systemui/theme/ThemeOverlayController.java
index 970ed1ca7..a1d8167b8 100644
--- a/packages/SystemUI/src/com/android/systemui/theme/ThemeOverlayController.java
+++ b/packages/SystemUI/src/com/android/systemui/theme/ThemeOverlayController.java
@@ -747,15 +747,6 @@ public class ThemeOverlayController implements CoreStartable, Dumpable {
         assignColorsToOverlay(mDynamicOverlay, DynamicColors.getFixedColorsMapped());
         // Custom Colors
         assignColorsToOverlay(mDynamicOverlay, DynamicColors.getCustomColorsMapped());
-
-        // Pure black theme: bottom sheets (USB mode chooser) and Pixel Launcher's app drawer with
-        // blur off are painted with this palette tone. The accent overlay is applied above the
-        // black theme overlay, so set it here.
-        if (LineageSettings.Secure.getIntForUser(mContext.getContentResolver(),
-                LineageSettings.Secure.BERRY_BLACK_THEME, 0, mUserTracker.getUserId()) == 1) {
-            mAccentOverlay.setResourceValue("android:color/system_accent2_800",
-                    TYPE_INT_COLOR_ARGB8, Color.BLACK, null);
-        }
     }
 
     private void assignColorsToOverlay(FabricatedOverlay overlay,
